### PR TITLE
[15.0][MIG] mrp_account: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT35757

ping @pedrobaeza

### CODE REVIEW

File: openupgrade_scripts/scripts/mrp_account/15.0.1.0/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---Models in module 'mrp_account'---
+---Fields in module 'mrp_account'---
+mrp_account  / account.analytic.account / bom_ids (one2many)            : NEW relation: mrp.bom
+mrp_account  / account.analytic.account / production_ids (one2many)     : NEW relation: mrp.production
+mrp_account  / account.analytic.account / workcenter_ids (one2many)     : NEW relation: mrp.workcenter
+mrp_account  / account.analytic.line    / category (False)              : NEW selection_keys: ['invoice', 'manufacturing_order', 'other', 'vendor_bill'], mode: modify
+mrp_account  / mrp.bom                  / analytic_account_id (many2one): NEW relation: account.analytic.account
+mrp_account  / mrp.production           / analytic_account_id (many2one): NEW relation: account.analytic.account, hasdefault: compute
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/mrp_account/15.0.1.0/upgrade_analysis_work.txt:10]: This is a computed stored, so it's going to be computed on module upgrade. Pre-create the column for avoiding this.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And rename the commit message to
```
[OU-ADD] mrp_account: Migration scripts
```

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**